### PR TITLE
Update Card, Nav and Navbar for V5

### PR DESCRIPTION
**Pull request by AnnMarieW**
Removed CardDeck
Removed CardColumns per [react-bootstrap issue#5944](https://github.com/react-bootstrap/react-bootstrap/issues/5944)

Updates for Nav and Navbar.  
Adds new NavbarCollapse component.  The react-bootstrap Collapse doesn't has a `navbar` prop

### CODE REVIEW

File: src/components/Collapse.js
Comment (tcbegley) [inline src/components/Collapse.js:85]: Rather than removing this and creating the new `NavbarCollapse` component, could we move the logic for that component into this file?

I think it might be possible to do something like

```jsx
const Component = navbar ? Navbar.Collapse : Collapse;

return <Component in={is_open} {...otherProps} />
```

then we can somewhat reduce the number of changes in the api. What do you think?

File: src/components/nav/Navbar.js
Comment (tcbegley) [inline src/components/nav/Navbar.js:111]: is `'bottom'` not also an option?

File: src/components/nav/NavbarSimple.js
Comment (tcbegley) [inline src/components/nav/NavbarSimple.js:5]: I think scoping the imports like `import Container from 'react-bootstrap'` allows for some optimisation of bundle size? I'm planning to go through everything when we're done and check all of these.

**Review by tcbegley — CHANGES_REQUESTED** [review]
Looks great, thanks @AnnMarieW!

Left a couple of inline comments, could you rebase on top of `react-bootstrap` once you've addressed then let's get this merged!

File: src/components/nav/Navbar.js
Comment (AnnMarieW) [reply src/components/nav/Navbar.js:111]: No, it's actually not an option in either [Bootstrap 4](https://getbootstrap.com/docs/4.0/components/navbar/#placement) or [Bootstrap 5](https://getbootstrap.com/docs/5.0/components/navbar/#placement).

I tried it in an app and if you use `sticky="bottom"` the class shows up in the DOM but it has no effect

File: src/components/Collapse.js
Comment (AnnMarieW) [reply src/components/Collapse.js:85]: good idea - I'll do that and remove the NavbarCollapse component

File: src/components/nav/Navbar.js
Comment (tcbegley) [reply src/components/nav/Navbar.js:111]: Got it, thanks! Seems strange that it's not just a boolean then 😛 

Let's leave it like this though to minimise breaking changes.

File: src/components/nav/NavbarSimple.js
Comment (tcbegley) [reply src/components/nav/NavbarSimple.js:5]: ^ note the unedited version of this comment made no sense 😅

Comment (AnnMarieW) [reply src/components/nav/NavbarSimple.js:5]: Oh, good, I was going to ask you about that...  I'll add the `/Container` - thanks!

File: src/components/nav/Navbar.js
Comment (AnnMarieW) [reply src/components/nav/Navbar.js:111]: So you mean it like `sticky: PropTypes.string,`   (and update the documentation so it doesn't mention bottom)
Right a boolean would be better.

Comment (tcbegley) [reply src/components/nav/Navbar.js:111]: Yeah leave it as a string since that's what it was before. Perhaps the maintainers of reactstrap and react-bootstrap made that choice in order to allow for `"bottom"` in future?

**Review by tcbegley — APPROVED** [review]
Amazing, thanks @AnnMarieW!

**Pull request merged**
